templates/nextflow-ecs-task-definition.j2

```diff
@@ -174,10 +174,34 @@ Resources:
           EFSVolumeConfiguration:
             FilesystemId: !Ref EfsFileSystemId
       ContainerDefinitions:
+        - Name: !Sub '${RedisContainerName}-CheckAOF'
+          Image: !Ref RedisContainerImage
+          Memory: 2000
+          Cpu: 0
+          Essential: false
+          EntryPoint:
+            - /bin/sh
+          # The `yes` command is needed because `redis-check-aof` has an
+          # interactive prompt and doesn't have an option to disable it
+          Command:
+            - -c
+            - "yes | redis-check-aof --fix appendonly.aof"
```

Comment:
i'm wondering if this is at all related? https://github.com/RediSearch/RediSearch/issues/394

Reply:
It seems related. An alternative fix is to enable `aof-load-truncated yes` in the `redis.conf` file, but that would involve more work (_e.g._ create the config file, copy it to EFS, and mount it to the Redis container). I'll keep that approach as a backup if this PR fails to address the issue consistently.